pkg/fqdn/cache.go

```diff
@@ -5,12 +5,13 @@ package fqdn
 
 import (
 	"encoding/json"
+	"maps"
 	"net/netip"
 	"regexp"
+	goslices "slices"
```

Comment:
I would have kept `slices` for the standard package and renamed the `cilium` one but maybe it is not part of this PR 😇

Reply:
thanks, I need to resolve go.mod conflict, so will incorporate your comment as well.